recipe/build.sh

```diff
@@ -16,5 +16,7 @@ cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
 
 
 make
+if [[ $(uname) == Linux ]]; then
```

Comment:
Is this cause the OSX builder doesn't have internet access? This test shouldn't be timing out. /cc @shahramn